### PR TITLE
Prisma adapter

### PR DESCRIPTION
**Pull request by noluckjustskill**

### CODE REVIEW

File: packages/moleculer-db-adapter-prisma/schema.prisma [hunk 1]
```diff
@@ -0,0 +1,17 @@
+generator client {
```
Comment (icebob) [inline packages/moleculer-db-adapter-prisma/schema.prisma:1]: Does this file exist only for example? Can we move it into the examples folder?

Comment (noluckjustskill) [reply packages/moleculer-db-adapter-prisma/schema.prisma]: yes, I think so

Comment (icebob) [reply packages/moleculer-db-adapter-prisma/schema.prisma]: After the moving, the tests failed: https://github.com/moleculerjs/moleculer-db/runs/7949047525?check_suite_focus=true

Comment (noluckjustskill) [reply packages/moleculer-db-adapter-prisma/schema.prisma]: This happened due to the fact that when using a custom schema file location, it is necessary to start the generation with an additional argument. The test is launched at a higher level and prehooks from packages are not processed. I propose to return the schema to the root and then the post-install hook will perform the generation.

Comment (noluckjustskill) [reply packages/moleculer-db-adapter-prisma/schema.prisma:1]: @icebob

**Comment by icebob** [comment]
Thanks, good job!

**Comment by icebob** [comment]
I can't release it because you took the project name on NPM. You should transfer the ownership for me on npmjs or if you want to publish it under your name, then, you should keep the code in a repo under your name, and not in this moleculerjs repo.

**Comment by noluckjustskill** [comment]
@icebob I invited you as maintainer 
image

**Comment by icebob** [comment]
Ok, but we have problem if you publish versions to NPM on different version numbers. I see you released a 0.1.17 3 hours ago.

**Comment by noluckjustskill** [comment]
Hi @icebob 
Can u update version in current repo please, next versions I will release via cross-repo PRs

**Comment by noluckjustskill** [comment]
HI @icebob 
I moved prisma-adapter to a separate repo: https://github.com/noluckjustskill/moleculer-db-adapter-prisma

**Pull request merged**